xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecTX3G.cpp

```diff
@@ -190,8 +186,8 @@ int CDVDOverlayCodecTX3G::Decode(DemuxPacket* pPacket)
         if (curRecord.endChar > textLength)
           curRecord.endChar = textLength;
 
-        bgnStyle.get()[curRecord.bgnChar] |= curRecord.faceStyleFlags;
-        endStyle.get()[curRecord.endChar] |= curRecord.faceStyleFlags;
+        bgnStyle.data()[curRecord.bgnChar] |= curRecord.faceStyleFlags;
```

Comment:
```suggestion
        bgnStyle[curRecord.bgnChar] |= curRecord.faceStyleFlags;
```

Why the extra `.data()`? (possibly due to search&replace?)

Reply:
updated, thanks.